src/network/networkprotocol.h

```diff
@@ -194,9 +194,11 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 		New network float format
 		ContentFeatures version 13
 		Add full Euler rotations instead of just yaw
+	PROTOCOL VERSION 38:
+		Incremental inventory sending mode
```

Comment:
maybe mention that unknown fields(?) in inv serialization now no longer throw an error